pavelib/assets.py

```diff
@@ -641,14 +641,28 @@ def restart_django_servers():
     ))
 
 
-def collect_assets(systems, settings):
+def collect_assets(systems, settings, debug=None, collectstatic_log=None):
     """
     Collect static assets, including Django pipeline processing.
     `systems` is a list of systems (e.g. 'lms' or 'studio' or both)
     `settings` is the Django settings module to use.
+    `debug` and `collectstatic_log` are used for determining where to
+       pipe collectstatic logging.
     """
+
+    # unless specified, collectstatic (which can be very verbose) pipes to /dev/null
+    collectstatic_stdout_str = "> /dev/null"
+    if debug:
+        # pipe to console
+        collectstatic_stdout_str = ""
+    if collectstatic_log:
+        # pipe to specified file
+        collectstatic_stdout_str = "> {output_file}".format(output_file=collectstatic_log)
```

Comment:
This is a bit odd, since the `debug` and `collectstatic_log` arguments collide.

Maybe just make it a `stdout_file_name` argument, and if it's `None`, then don't redirect `stdout`?